Jenkinsfile

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env groovy
+
+@Library('sec_ci_libs@v2-latest') _
+
+def master_branches = ["master", ] as String[]
+
+pipeline {
+  agent {
+    dockerfile {
+      args  '--shm-size=2g'
+    }
+  }
+
+  environment {
+    JENKINS_VERSION = 'yes'
+  }
+
+  options {
+    timeout(time: 1, unit: 'HOURS')
+  }
+
+  stages {
+    stage('Authorization') {
+      steps {
+        user_is_authorized(master_branches, '8b793652-f26a-422f-a9ba-0d1e47eb9d89', '#frontend-dev')
+      }
+    }
+
+    stage('Initialization') {
+      steps {
+        ansiColor('xterm') {
+          retry(2) {
+            sh '''npm install'''
+          }
+        }
+      }
+    }
+
+    stage('Unit Test') {
+      steps {
+        ansiColor('xterm') {
+          sh '''npm run test'''
+        }
+      }
+    }
+
+    stage('Build') {
+      steps {
+        ansiColor('xterm') {
+          sh '''npm run dist'''
+        }
+      }
+
+      post {
+        always {
+          archiveArtifacts 'dist/*'
+        }
+      }
+    }
+
+    stage('Deploy'){
```

Comment:
We should only deploy when a PR is merged so probably a separated hook. Seems like here will deploy everytime there is a PR successful merged or not merged, right @DanielMSchmidt?